modules/conflict_resolver/help/conflict_resolver.md

```diff
@@ -1,8 +1,9 @@
 # Conflict Resolver
 
-The Conflict Resolver module enables studies to identify and resolve typos and discrepancies that arise between initial data entry and double data entry forms. Unresolved and Resolved conflicts are displayed on two separate tabs of the module.
-Unresolved conflicts are displayed by default upon accessing the module. Selection filters can be applied to search for a particular subject and/or instrument. Clicking "Show Data" after entering selection filter criteria, to display data results. Click the “Clear Form” button to reset all Selection Filters.  The Selection Filters panel can be hidden by clicking the upward arrow in the panel title bar.
-By default, unresolved data entry conflicts appear as “Unresolved” under the “Correct Answer” column, until a user manually resolves the data discrepancy.
-To resolve a conflict, click in the “Correct Answer” column dropdown select for a given data entry item. This dropdown-select will show two options: these are the data values that were entered in each of the initial and the double data entry forms. Select one of these options as the correct answer, after cross-checking against original data records (e.g. paper forms) for accuracy.
-To finalize this change and to successfully resolve the conflict, users must click “Save” located at the bottom of the results table.
-After refreshing the page, this newly resolved conflict will move to the Resolved Conflicts tab in the module.
+This module is designed to identify and resolve data conflicts in a study. LORIS uses double data entry, which means you may be asked to enter critical data twice in duplicate instrument forms. The Conflict Resolver will flag any discrepancies between the two entries (called *unresolved conflicts*) and allow you to fix—resolve—them easily. 
```

Comment:
Same here, the conflict resolver changes should be separated.

Reply:
I don't understand your comment

Reply:
This PR shouldn't include the conflict_resolver help text file, just dicom_archive, is what i'm pretty sure he means. 

@knoll-alex if you get stuck removing the extra file from this branch/PR, let me know

Reply:
@driusan The conflict resolver file was accidentally included in multiple PRs. I commented mroe ind etail on another one.

Reply:
thanks, i removed the file. not sure what happened there